staging/src/k8s.io/api/node/v1alpha1/types.go

```diff
@@ -115,3 +115,66 @@ type RuntimeClassList struct {
 	// Items is a list of schema objects.
 	Items []RuntimeClass `json:"items" protobuf:"bytes,2,rep,name=items"`
 }
+
+// +k8s:conversion-gen:explicit-from=net/url.Values
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NodeLogOptions is the query options for a Node's logs REST call.
+type NodeLogOptions struct {
```

Comment:
n.b. we also have `LoggingOptions` on a number of components that use the component-base logging machinery... I might suggest a different name for this that's more clearly related to the KEP, like `NodeLogViewerOptions`

Reply:
not opposed to this, but I want to mention that this was named analogously to the `PodLogOptions` API. `NodeLogViewerOptions` sgtm too though.

Reply:
renamed to NodeLogViewerOptions